src/lesson1/L1Task5.java

```diff
@@ -0,0 +1,9 @@
+package lesson1;
+
+public class L1Task5 {
+    //Створити метод перетворення дюймів в сантиметри
+    public static double convertToCentimeter(double inch) {
+        double result = inch*2.56;
+        return result;
```

Comment:
можна зразу повертати результат обчислень, ну і форматування
```suggestion
        return inch * 2.56;
```